uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/test/shared/models/instances/toil-leave-request-instance_test.js

```diff
@@ -114,5 +115,22 @@ define([
         expect(instance.toAPI).toHaveBeenCalled();
       });
     });
+
+    describe('updateDuration()', function () {
+      beforeEach(function () {
+        instance = TOILRequestInstance.init({}, false);
+        instance.toilDurationHours = 1
```

Comment:
If we also add a `minute`here and perform the test, we can make sure the `minute` is taken into account in conversion process.